File: app/src/main/java/com/cyinfo/android/study/MainActivity.java
package com.cyinfo.android.study;

import android.os.Bundle;
import android.view.View;
import android.view.Window;
import android.widget.Button;
import android.widget.Toast;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.FragmentActivity;

import butterknife.BindView;
import butterknife.ButterKnife;

public class MainActivity extends FragmentActivity {


    @BindView(R.id.true_button)
    public Button trueButton;

    @BindView(R.id.false_button)
    public Button falseButton;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        requestWindowFeature(Window.FEATURE_NO_TITLE);
        setContentView(R.layout.activity_main);
        ButterKnife.bind(this);
        trueButton.setOnClickListener((view) ->
                Toast.makeText(MainActivity.this, "点击确定按钮", Toast.LENGTH_SHORT).show());
    }
}

File: app/src/main/java/com/cyinfo/android/study/MainActivity.kt
package com.cyinfo.android.study

import android.app.Activity
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.view.ViewGroup
import android.widget.Button
import android.widget.ImageView
import android.widget.LinearLayout
import android.widget.TextView

class MainActivity : Activity() {

    private var images =
        intArrayOf(R.drawable.java, R.drawable.javaee, R.drawable.swift, R.drawable.ajax, R.drawable.html)
    private var currentImg = 0

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        var main = findViewById<LinearLayout>(R.id.root)
        var image = ImageView(this)
        main.addView(image)
        image.setImageResource(images[0])
        image.setOnClickListener {
            image.setImageResource(images[++currentImg % images.size])
        }
    }
}
